src/hw_isolation_record/manager.cpp

```diff
@@ -544,6 +549,8 @@ void Manager::handleHostIsolatedHardwares()
     // Delete all the D-Bus entries if no record in their persisted location
     if ((records.size() == 0) && _isolatedHardwares.size() > 0)
     {
+        // Clean up all entries association before delete.
```

Comment:
this method called before delete?

Reply:
we are clearing all the _isolatedHardwares list then how it will be useful when it is called in delete all?

Reply:
Yes, Before deleting the D-Bus entries objects we need to set the `Enabled` D-Bus property that's present in the inventory item (say for example Core/DIMM) as `true` because that property is mapped for the guard record and that will be changed as false during creation so if we deleted, it needs to be set as `true`